recipes/intreehooks/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.0" %}
+
+package:
+  name: intreehooks
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/i/intreehooks/intreehooks-{{ version }}.tar.gz
+  sha256: 87e600d3b16b97ed219c078681260639e77ef5a17c0e0dbdd5a302f99b4e34e1
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
I don't think we can have noarch when the recipe uses the compiler. What am I missing?

Reply:
Wait, I believe that was a mistake on my part. Let me double check the package.